### ISSUE TITLE
IAFD - Unicode issue

### ISSUE THREAD

**Issue by CodyBerenson** [CodyBerenson/PGMA-Modernized#97]
Certain accented characters are not handled by IAFD.  

![image](https://user-images.githubusercontent.com/40373801/117439435-d4cd0980-af00-11eb-833a-d617d6731c95.png)

**Comment by jfreiman** [comment]
I have this issue too.  Particularly with the single-outside quote  ’ symbol which appears more frequently in titles than I would have thought.  I have gone through my scene and feature collections and replaced all of the ’ symbols with a simple single quote ' instead -- I used the _Bulk Rename Utility_ for Windows which worked quite well.
After I did that I realized I should have searched through my collections metadata and manually replaced the symbol manually -- which I ultimately did.  It took more time this way, but it saves me doing this again if I have to edit or move the file at a later point.
